### PR TITLE
fix: seo with yoast disabled

### PR DESCRIPTION
**Pull request by nicholasio**

### CODE REVIEW

File: packages/next/src/rsc/data/seo.ts [hunk 1]
```diff
@@ -17,9 +17,8 @@ const { all: merge } = deepmerge;
 
 function isPostEntity(data: Entity | QueriedObject): data is PostEntity {
 	return (
-		typeof (data as PostEntity).yoast_head !== 'undefined' &&
-		typeof (data as PostEntity).yoast_head_json !== 'undefined' &&
```
Comment (Antonio-Laguna) [inline packages/next/src/rsc/data/seo.ts:21]: Is this removal intentional?

**Review by Antonio-Laguna — COMMENTED** [review]
This LGTM. Not sure removing the check for `yoast_head` is intentional for this PR though!

File: packages/next/src/rsc/data/seo.ts (hunk 1 above)
Comment (nicholasio) [reply packages/next/src/rsc/data/seo.ts:21]: Yes! I want this to work even without Yoast, so I should not rely on it for detecting post entity object. In the future I also want to support other plugins.
